documentation/Using-Azure-TestFramework.md

```diff
@@ -150,7 +154,7 @@ Setting `Environment` in the connection string will result in the following defa
 
 ##### Environment = Dogfood
 
-	AADAuthUri = "https://login.windows-ppe.net";
+		AADAuthUri = "https://login.windows-ppe.net";
```

Comment:
@shahabhijeet it looks like `AADAuthUri` has a different indentation in a couple of the below environments. Can you make sure they are lined up?